Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -75,16 +75,52 @@ users to adapt their environments or deployment automation to the new standards
+   An expression used to select test cases by referring to the tags that must (or must not) be present.
+6. _Target_:
+   A named collection of test cases specified using selector expressions.
+   Ultimately, the certification of a subject always depends on a designated "main" target; all its test cases must be passed for the certificate to be awarded.
+   Further targets can be used to report on optional aspects of the certificate, such as particularly good security and encryption measures.
+7. _(Certificate-scope) version_:
+   A collection of modules and a collection of targets, one of them being "main".
+   Note that a collection of modules can again be construed as a (larger) module. We opt to use one module per standard version, as mentioned above, in order to make commonalities between certificate-scope versions explicit.
+8. _Certificate scope_:
+   A list of certificate-scope versions.
+
+Having introduced these concepts, we can now get even more precise by defining the actual specification in YAML format.
+
 ## SCS Certification YAML
 
 Each certificate scope is recorded in a dedicated YAML file, e.g. `scs-open-kaas.yaml`.
+For an example of such a file, see
+[scs-compatible-iaas.yaml](https://github.com/SovereignCloudStack/standards/blob/main/Tests/scs-compatible-iaas.yaml) or
+[scs-compatible-kaas.yaml](https://github.com/SovereignCloudStack/standards/blob/main/Tests/scs-compatible-kaas.yaml).
 
 The certification YAML _MUST_ contain the following keys:
 
 | Key        | Type          | Description                                          | Example                                                                                         |
 | ---------- | ------------- | ---------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
-| `name`     | String        | Full name of this certificate scope                  | _SCS Open KaaS_                                                                                 |
+| `name`     | String        | Full name of this certificate scope                  | `SCS-open KaaS`                                                                                 |
 | `url`      | String        | Valid URL to the latest raw version of this document | `https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Tests/scs-open-kaas.yaml` |
+| `modules`  | Array of maps | List of module descriptors (described below)         | (see below)                                                                                     |
+| `timeline` | Array of maps | List of timeline entries (described below)           | (see below)                                                                                     |
 | `versions` | Array of maps | List of version descriptors (described below)        | (see below)                                                                                     |
```

Comment:
What about `uuid`, I noticed this in the YAML files?

Reply:
Good point, this needs to be mentioned here.

Reply:
see 5bf0229f6cd4d31f54a6b52c5595dbcbeffb394f